frontend/src/components/VirtualList/Config.ts

```diff
@@ -237,6 +238,8 @@ export const IstioTypes: { [type: string]: IstioConfigType } = {
     url: 'requestauthentications',
     badge: PFBadges.RequestAuthentication
   },
+  // TODO should be merged with k8sreferencegrant
+  referencegrant: { name: 'ReferenceGrant (K8s)', url: 'k8sreferencegrants', badge: PFBadges.K8sReferenceGrant },
```

Comment:
Is this still needed here?

Reply:
yes, this is here because of a design issue of mixing K8s and Istio objects, probably we need a separate issue to fix this